app/scripts/extensions/nav/dropdownMobile.js

```diff
@@ -31,16 +31,42 @@ angular.module('openshiftConsole')
               '</a>',
             '</li>'
           ].join('')
-        }, {
-          type: 'dom',
-          node: _.template([
-            '<li>',
-              '<a href="logout">',
-                '<span class="pficon pficon-user fa-fw" aria-hidden="true"></span>',
-                'Log out <span class="username"><%= userName %></span>',
-              '</a>',
-            '</li>'
-          ].join(''))({userName: (user ? (user.fullName || user.metadata.name) : "") })
         }];
+
+        var launcherApps = Constants.APP_LAUNCHER_NAVIGATION;
+        var appDropdownItems = [];
```

Comment:
You should be able to push the items onto `dropdownItems` directly and not need this local var.